packages/schemas/src/db-entries/application.ts

```diff
@@ -15,15 +15,15 @@ export type ApplicationDBEntry = {
   name: string;
   type: ApplicationType;
   oidcClientMetadata: OidcClientMetadata;
-  createdAt: number;
+  createdAt?: number;
```

Comment:
why createdAt is optional?
why not required?

Reply:
why not add updatedAt?

Reply:
This is an auto generate file.
`createdAt` is optional because it has a default value `now()`

Reply:
> 

ok